Apps/W1/DynamicsGPHistoricalData/app/src/Pages/HistPayablesApplyList.Page.al

```diff
@@ -0,0 +1,93 @@
+namespace Microsoft.DataMigration.GP.HistoricalData;
+
+page 41021 "Hist. Payables Apply List"
+{
+    ApplicationArea = All;
+    Caption = 'Historical Payables Apply Lines';
+    PageType = ListPart;
+    Editable = false;
+    DeleteAllowed = false;
+    InsertAllowed = false;
+    LinksAllowed = false;
+    ModifyAllowed = false;
+    SourceTable = "Hist. Payables Apply";
+    CardPageId = "Hist. Payables Apply";
+    UsageCategory = None;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(General)
+            {
+                field("Apply To Document No."; Rec."Apply To Document No.")
+                {
+                    ToolTip = 'Specifies the value of the Apply To Document No. field.';
+                    Visible = not IsInvoice;
```

Comment:
Application area is missing on the fields, see if caption is needed (if it can take the caption from table without the warning)

Reply:
Same comment on all pages

Reply:
As of BC22, the field level ApplicationArea property is inherited from the page if it isn't explicitly set on the field. I prefer this because it reduces redundancy.

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-applicationarea-property

I don't have any warnings and the captions show up fine on the pages.